app/models/review.rb

```diff
@@ -15,9 +15,10 @@ class Review < ActiveRecord::Base
     greater_than_or_equal_to: 1, less_than_or_equal_to: 5 }
   validates :weather_rating, numericality: { only_integer: true,
     greater_than_or_equal_to: 1, less_than_or_equal_to: 5 }
-  validates :user, presence: true
-  validates :city, presence: true
-
+  validates :user, presence: true,
+    uniqueness: {scope: :city_id}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.